cmake/CompilerFlags.cmake

```diff
@@ -23,10 +18,16 @@ set(GCC_DEBUG
 set(GCC_RELEASE -O3)  # Level 3 optimisation. Could also consider -fpack-derived
 
 # Intel 
+set(INTEL_BASE
+    -stand f08          # Fortran standard set to 2008
```

Comment:
Comment's wrong for this, `stand` warns w.r.t. use of non-standard fortran. See [here](https://www.intel.com/content/www/us/en/docs/fortran-compiler/developer-guide-reference/2023-2/stand.html)

This should not be a base flag

Reply:
Thanks for pointing out the correct meaning of `-stand`.

The reason I added it as an intel base flag is to correspond to the original `-std='f2008'` flag for GCC. The other flags in `INTEL_BASE` also follow this intention.